tools/ledger/src/test/java/org/apache/bookkeeper/tools/cli/commands/bookie/ConvertToInterleavedStorageCommandTest.java

```diff
@@ -0,0 +1,127 @@
+public class ConvertToInterleavedStorageCommandTest extends BookieCommandTestBase {
+
+    @Rule
+    private TemporaryFolder folder = new TemporaryFolder();
+
+    public ConvertToInterleavedStorageCommandTest() {
+        super(3, 0);
+    }
+
+    @Override
+    public void setup() throws Exception {
+        super.setup();
+
+        createTmpFile();
+        PowerMockito.whenNew(ServerConfiguration.class).withNoArguments().thenReturn(conf);
+        PowerMockito.whenNew(ServerConfiguration.class).withParameterTypes(AbstractConfiguration.class)
+            .withArguments(eq(conf)).thenReturn(conf);
+
+        DiskChecker diskChecker = mock(DiskChecker.class);
+        whenNew(DiskChecker.class).withArguments(conf.getDiskUsageThreshold(), conf.getDiskUsageWarnThreshold())
+            .thenReturn(diskChecker);
+
+        LedgerDirsManager ledgerDirsManager = mock(LedgerDirsManager.class);
+        whenNew(LedgerDirsManager.class).withParameterTypes(ServerConfiguration.class, File[].class, DiskChecker.class)
+            .withArguments(conf, conf.getLedgerDirs(), diskChecker).thenReturn(ledgerDirsManager);
+        when(ledgerDirsManager.getAllLedgerDirs()).thenReturn(getFileList());
+
+        DbLedgerStorage dbStorage = mock(DbLedgerStorage.class);
+        whenNew(DbLedgerStorage.class).withNoArguments().thenReturn(dbStorage);
+        when(dbStorage.getActiveLedgersInRange(anyLong(), anyLong())).thenReturn(this::getLedgerId);
+
+        LedgerCache interleavedLedgerCache = mock(LedgerCache.class);
+        doNothing().when(interleavedLedgerCache).flushLedger(anyBoolean());
+
+        InterleavedLedgerStorage interleavedLedgerStorage = mock(InterleavedLedgerStorage.class);
+        whenNew(InterleavedLedgerStorage.class).withNoArguments().thenReturn(interleavedLedgerStorage);
+        doNothing().when(interleavedLedgerStorage).flush();
+        doNothing().when(interleavedLedgerStorage).shutdown();
+        when(interleavedLedgerStorage.getLedgerCache()).thenReturn(interleavedLedgerCache);
+    }
+
+    private Iterator<Long> getLedgerId() {
+        Vector<Long> longs = new Vector<>();
+        LongStream.range(0L, 10L).forEach(longs::add);
+        return longs.iterator();
+    }
+
+    private List<File> getFileList() {
+        List<File> files = new LinkedList<>();
+        files.add(folder.getRoot());
+        return files;
+    }
+
+    private void createTmpFile() {
+        try {
+            folder.newFile("ledgers");
+            folder.newFile("locations");
+            System.out.println(folder.getRoot().getAbsolutePath());
+        } catch (IOException e) {
+            throw new UncheckedExecutionException(e.getMessage(), e);
+        }
+    }
+
+    @Test
+    public void testConvertToInterleavedStorageCommand() {
+        ConvertToInterleavedStorageCommand cmd = new ConvertToInterleavedStorageCommand();
+        Assert.assertTrue(cmd.apply(bkFlags, new String[] { "" }));
```

Comment:
I think it would be better to verify the command is executing as expected. e.g. verify if those methods are called

Reply:
ok, i fix it. PTAL

Reply:
Am I fix right?